### DISCUSSION TITLE
Lab2 - Why I can not find the utilization information in the synthesis report?

### DISCUSSION THREAD

**Discussion by michael61112** [bol-edu/HLS-SOC-Discussions#111] [Q&A]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/ed97a662-b831-4075-843d-88af5d277af6)
我記得在lab1的時候可以在synthesis report可以找到utilization的資訊，這次lab2好像找不到相關資訊?也沒有報error

**Comment by edwardchang9** [comment]
你說的utilization的資訊是指什麼? 右下角已經有你用的BRAM、DSP、FF和LUT的資訊了

**Reply by michael61112** [reply]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/e536018a-8e34-41a0-a68b-73caf3e7aed7)

之前看會有類似這個表出現，所以不是要看這個表喔?
